src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    protected static Random random = new Random();
```

Comment:
The `random` field should not be static. According to the checklist, static methods and fields are generally considered a bad practice. Consider making it a non-static field.